tests/unit/cloudformation/test_connection.py

```diff
@@ -444,7 +444,8 @@ def test_describe_stacks(self):
         self.assertEqual(stack.stack_id, 'arn:aws:cfn:us-east-1:1:stack')
         self.assertEqual(stack.stack_status, 'CREATE_COMPLETE')
         self.assertEqual(stack.stack_name, 'MyStack')
-        self.assertEqual(stack.stack_name_reason, None)
+        self.assertEqual(stack.stack_name_reason, '')
+        self.assertEqual(stack.stack_status_reason, '')
```

Comment:
Can we update the response body in this test to have a StackStatusReason so we can verify we're parsing the expected value?